File: programmers_day.c
/**
*
* Fun code for programmers day
* Programmers day is the 256th day of the year
* https://en.wikipedia.org/wiki/Day_of_the_Programmer
* Iván Chavero <ichavero@chavero.com.mx>
*
**/

#include <stdio.h>
#include <time.h>

void main(void) {
    time_t t = time(NULL);
    struct tm *today = localtime(&t);
    if(today->tm_yday == 255) {
        printf("Happy Programmers day!!\n");
    }
    else {
        printf("Happy shitty normal day!!\n");
    }
}

File: programmers_day.py
#!/bin/python
# -*- coding: utf-8 -*-

# Fun code for programmers day
# Programmers day is the 256th day of the year
# https://en.wikipedia.org/wiki/Day_of_the_Programmer
# Iván Chavero <ichavero@chavero.com.mx>

import time

yday = time.localtime().tm_yday

if yday == 256:
    print "Happy programmers day!!";
else: 
    print "Happy shitty normal day!!";

